### ISSUE TITLE
Steam prevents menu shutdown/reboot

### ISSUE THREAD

**Issue by TheSHEEEP** [ValveSoftware/steam-for-linux#7005]
#### Your system information

* Steam client version (build number or date): Mar 26 2020, at 22:46:20, package v: 1585264767
* Distribution (e.g. Ubuntu): Ubuntu 18.04, Manjaro KDE, others...
* Opted into Steam client beta?: [Yes/No] Yes
* Have you checked for system updates?: [Yes/No] Yes

#### Please describe your issue in as much detail as possible:
When trying to shut down or reboot my computer using menu options, only Steam gets closed. A second attempt will then actually work.

This happens across all of my computers (various different hardware) and across all distributions I use.
Basically the same as #3301 - which never should have been closed, but seems to have been forgotten.

Shutting down via console "shutdown -h 0" works just fine, btw. Not really sure what the difference is.

#### Steps for reproducing this issue:

1. Try to shut down / reboot your PC via menu.
2. Notice that only Steam closes.
3. Need to shut down again - it will work now.

**Comment by Timeroot** [comment]
Having this issue as well. Basically the same as #3301. Happens on both the Steam Client Beta, and not (on the latest versions available today, May 8th 2020).

Running Ubuntu 18.04 with Cinnamon.

**Comment by dajusc** [comment]
On my Linux Mint Cinnamon it's like this since February 2018. See #5379, which sadly never got fixed. I have to use a desktop shortcut with "killall steam", or my shutdown takes forever.

**Comment by TTimo** [comment]
The comments on https://github.com/ValveSoftware/steam-for-linux/issues/5379 suggest that this is specific to Cinnamon DE - or more likely it's Window Manager, a fork of GNOME3's Window Manager according to wikipedia? Are you using Wayland at all, or is this on XOrg?

**Comment by TheSHEEEP** [comment]
I'm on Manjaro with KDE Plasma. I don't think there's much in common with GNOME3's Window Manager?

In any case, though, Steam is the only application showcasing this behavior (as far as I know). I truly wonder what the reason for this oddity is.
